docs/Configuration/yaml-files.md

```diff
@@ -630,8 +630,7 @@ org_settings:
 
 - `enable_vulnerabilities_webhook` (default: `false`)
 - `destination_url` is the URL to `POST` to when the condition for the webhook triggers (default: `""`).
-- `days_count` is the number of days that hosts need to be offline to count as part of the percentage (default: `0`).
```

Comment:
this looks like some copy/pasta from the `host_status_webhook` docs above it -- not relevant for the vulnerabilities webhook